tests/Unit/HtmlProcessor/AbstractHtmlProcessorTest.php

```diff
@@ -760,25 +755,23 @@ static function (array $dataset) {
     }
 
     /**
-     * @return string[][] Each dataset has three elements in the following order:
+     * @return array<string, array{0:string, 1:string, 2:string}>
+     *         Each dataset has three elements in the following order:
      *         - HTML with non-XML self-closing tags (e.g. "...<br>...");
      *         - The equivalent HTML with XML self-closing tags (e.g. "...<br/>...");
      *         - The name of a self-closing tag contained in the HTML (e.g. "br").
```

Comment:
Multiline `@return` DocBlock entries are generally not indented from what I've seen, as there is only ever one `@return` and there will be blank lines above and below if necessary.  Though I accept this is how it was already formatted; it's a case of 'since you're here you could change it'.